automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/WorkspaceDataPage.scala

```diff
@@ -117,7 +117,7 @@ class ImportMetadataModal(implicit webDriver: WebDriver) extends OKCancelModal("
   def importFile(file: String): Unit = {
     importFromFileButton.doClick()
     fileUploadInput.selectFile(file)
-    await condition (find(previewText).get.underlying.getText.contains("entity"))
+    await condition (previewText.element.underlying.getText.contains("entity"))
```

Comment:
I don't think this will help. `element` will throw `TestFailedException` if no element is found which I doubt will be treated any differently than `NoSuchElementException`.

Here's an alternative:
```
await condition find(previewText).exists(_.underlying.getText.contains("entity"))
```

Reply:
I realized it wasn't right last night. I made new change before saw your comment just now. I think my new change match what you recommended. right?
`await condition previewText.findElement.exists(_.underlying.getText.contains("entity"))`

Reply:
Mostly. The only difference is that using `find` (our override in `WebBrowserUtil`) has the benefit of being tolerant of `StaleElementReferenceException` which can sometimes be a fact of life when testing a react application.